.github/workflows/_binary-build-linux.yml

```diff
@@ -67,7 +67,7 @@ on:
 
 jobs:
   build:
-    runs-on: linux.4xlarge
+    runs-on: linux.12xlarge
     timeout-minutes: 270
```

Comment:
Should we reduce timeout minutes to 120 for example?

Reply:
Sure, we can do so